scripts/exregional_run_analysis.sh

```diff
@@ -348,6 +348,8 @@ niter2=50
 lread_obs_save=.false.
 lread_obs_skip=.false.
 if_model_dbz=.false.
+ii_use_2mq4b=2
+ii_use_2mt4b=1
```

Comment:
See a previous comment on this. Also suggest to put them in the config files.

Reply:
@guoqing-noaa 

Correction: this is analysis type dependent, needs to be here. My GSI doesn't support (2,1) options for the two parameters.

Reply:
@guoqing-noaa 
The two parameters had been defined in config file: 
i_use_2mQ4B=0
i_use_2mT4B=0

They need to assign different values for meteorological DA or SD DA, so they are also in this shell script.